packages/flutter_tools/lib/src/commands/upgrade.dart

```diff
@@ -219,34 +218,6 @@ class UpgradeCommandRunner {
     }
   }
 
-  /// Attempts to reset to the last non-hotfix tag.
```

Comment:
Removing this as from this point forward, upgrades will be possible with mere fast forward.

Reply:
Woot!